### ISSUE TITLE
[Accessibility] Expandable sections are not keyboard focusable and do not communicate expansion state

### ISSUE THREAD

**Issue by PeterSaathoff-Harshfield** [ampproject/amp.dev#4508]
Environment:
Any OS/browser and screen reader (such as VoiceOver, NVDA, or ChromeVox)

Steps to reproduce:
1. Open https://amp-dev-staging.appspot.com/page-experience/
2. Enter a valid URL and press Analyze
3. Press tab to move keyboard focus to the expandable sections under the "Take action. Improve your AMP site" heading

Expected:
It should be possible to operate the expandable sections with the keyboard.

Actual:
The expandable sections are not keyboard focusable.

4. Turn on a screen reader
5. Navigate with the screen reader to the buttons
6. Activate the buttons to expand and collapse the sections

Expected:
When the section is expanded, the screen reader should announce the expanded state, and the same for collapsed state.

Actual:
It is possible to activate the buttons with screen readers, but there is no expanded or collapsed state, so users don't get the information that is visually conveyed by the disclosure arrow at the right side.

[Info on implementing accessible expandable sections](https://www.accessibility-developer-guide.com/examples/sensible-aria-usage/expanded/)

![image](https://user-images.githubusercontent.com/30604285/92979528-dc0e4700-f447-11ea-83e9-ee26db64df48.png)
